functions/Copy-DbaDbTableData.ps1

```diff
@@ -94,6 +94,9 @@ function Copy-DbaDbTableData {
     .PARAMETER BulkCopyTimeOut
         Value in seconds for the BulkCopy operations timeout. The default is 5000 seconds.
 
+    .PARAMETER CommandTimeOut
+        Value in seconds for the reader command operations timeout. The default is 0 seconds (no timeout).
```

Comment:
This has an effect on more than the reader command since we pass the connection string with the command timeout set to the BulkCopy object. Bulk copy operation will still adhere to the command timeout, if the network packets hit the command timeout then that bulk copy will fail as well.

```suggestion
        Gets or sets the wait time (in seconds) before terminating the attempt to execute the reader and bulk copy operation. The default is 0 seconds (no timeout).
```

Reply:
Thanks for the suggestion, I liked it. Will change now